services/server/config/config.go

```diff
@@ -54,6 +59,42 @@ type Config struct {
 	md toml.MetaData
 }
 
+// GetVersion returns the config file's version
+func (c *Config) GetVersion() int {
+	if c.Version == 0 {
+		return 1
+	}
+	return c.Version
+}
+
+// ValidateV2 validates the config for a v2 file
+func (c *Config) ValidateV2() error {
+	if c.Version != 2 {
+		return nil
+	}
+	for _, p := range c.DisabledPlugins {
+		if len(strings.Split(p, ".")) < 4 {
```

Comment:
`4` means `plugins.io.containerd.vx.xxx` right? 
I think we can add the documentation for this or make error more clear. what do you think?

Reply:
updated